Formula/kakoune.rb

```diff
@@ -0,0 +1,26 @@
+class Kakoune < Formula
+  desc "Selection-based modal text editor"
+  homepage "https://github.com/mawww/kakoune"
+  url "https://github.com/mawww/kakoune/releases/download/v2018.04.13/kakoune-2018.04.13.tar.bz2"
+  sha256 "cd8ccf8d833a7de8014b6d64f0c34105bc5996c3671275b00ced77996dd17fce"
+
+  depends_on "asciidoc" => :build
+  depends_on "docbook-xsl" => :build
+
+  if MacOS.version <= :el_capitan
+    depends_on "gcc"
+    fails_with :clang
```

Comment:
Has this been reported?

Reply:
Yeah, upstream are aware. It's because they're using C++ features not available on El Capitan's clang. I don't think they have any plans to stop using those features, though.

Reply:
OK. Can you please add a Ruby comment to that effect in the formula or use the weird `fails_with do ... cause` DSL and put the comment in `cause`?

Reply:
The build number is `build 800` if you do the latter.